crates/core/src/operations/optimize.rs

```diff
@@ -251,10 +263,38 @@ impl<'a> OptimizeBuilder<'a> {
         self.min_commit_interval = Some(min_commit_interval);
         self
     }
+
+    /// Commit Writes
+    pub fn with_commit_writes(mut self, commit_writes: bool) -> Self {
+        self.commit_writes = commit_writes;
+        self
+    }
+}
+
+impl<'a> OptimizeBuilder<'a> {
+    /// Commit writes after processing
+    pub async fn commit_writes(
+        self,
+        commit_info: CommitContext
+    ) -> DeltaResult<DeltaTable> {
+        let _ = commit(
```

Comment:
what's the reason for `let _ =`?

Reply:
Updated to use `?` and propagate the error